NEXT_CHANGES.rst

```diff
@@ -15,6 +15,48 @@ links to relevant Issues, Discussions, and PR's on github with the following for
 ixdat 0.2.6
 ===========
 
+API changes
+-----------
+
+Readers
+^^^^^^^
+- A function ``fix_WE_potential`` has been added to the ``biologic`` module. This
+  corrects the mistake in some .mpt files that are exported with zeros for "<Ewe>/V".
+
+Techniques
+^^^^^^^^^^
+- MS measurements now have access to the powerful ``spectro_inlets_quantification`` (siq)
+  package as a plugin. See `siq's documentation <https://github.com/SpectroInlets/spectro_inlets_quantification/blob/main/docs/source/theory/quant_ms.md>`_
```

Comment:
Hm... this confused me a bit, but it looks like this comment is outdated and I have fixed it since: https://github.com/ixdat/ixdat/pull/116/files
(make sure to select changes from "all commits" in top left, easy to miss!)